src/main.js

```diff
@@ -11,7 +11,7 @@ module.exports = {
     this.subscriptions = new CompositeDisposable()
     this.active = true
     this.worker = null
-    this.scopes = ['source.js', 'source.jsx', 'source.js.jsx', 'source.babel', 'source.js-semantic']
+    this.scopes = atom.config.get('linter-eslint.scopes')
```

Comment:
Maintaining the scopes list isn't _quite_ this simple. Your current code will only update on launching Atom, since that is (generally) the only time the `activate()` function is called.

You'll need to update this to code like [this](https://github.com/AtomLinter/linter-proselint/blob/483fa1d7c65f7f83a1995a7aa4b546a734378bab/lib/index.js#L12-L17), so whenever the setting is changed, the scopes array is update (_not_replaced!) with the new values.